docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

```diff
@@ -91,7 +90,7 @@ footer_links_extracustom_text = Custom text
 footer_links_extracustom_url = http://your.custom.site
 ```
 
-Here is the same example using environment variables instead of the custom.ini or grafana.ini file.
+The following example show how to configure custom branding using environment variables instead of the `custom.ini` or `grafana.ini` files.
```

Comment:
```suggestion
The following example shows configuring custom branding using environment variables instead of the `custom.ini` or `grafana.ini` files.
```